$OpenBSD: patch-src_main_cpp,v 1.1 2015/01/21 02:26:00 bcallah Exp $

Upstream forgot that cmd is first under #if defined(linux) ...

--- src/main.cpp.orig	Mon Jan 12 00:20:21 2015
+++ src/main.cpp	Mon Jan 12 00:23:56 2015
@@ -330,7 +330,7 @@ int main(int argc,char *argv[])
 #endif
 
 	// Test if iBus is running (this is used to fix the bug with xim and iBus input managers)
-	cmd="pgrep -u " + FXStringVal(getuid()) + " ibus-daemon";
+	FXString cmd="pgrep -u " + FXStringVal(getuid()) + " ibus-daemon";
 	FXuint pid = FXUIntVal(getCommandOutput(cmd));
 	if (pid == 0)
 		run_ibus=false;
